### PR TITLE
adapt license header

### LINKED ISSUE #31: Licence header in every file
Other plugins start the license header with
`// This file is part of Moodle - https://moodle.org/`

So far in this repo I've been using
`// This file is part of the ocis repository for Moodle - http://moodle.org/`

What do we want to do in future?

CC @PhMemmel @phil-davis @dj4oC

### PR DESCRIPTION
**Pull request by individual-it**
make license header display all needed information

moodle style is stict about the length of the lines and the content of the third line, that's why I've placed the contributors at the bottom.
Alternatively we could add the contributors into the PHPDoc part with `@author` or `@copyright` but that does not work so well, because the author is more of a person and not all contributor (orgs) have copyright

fixes #31

### CODE REVIEW

File: classes/configuration_exception.php [hunk 1]
```diff
@@ -14,6 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with the ocis repository for Moodle.  If not, see <http://www.gnu.org/licenses/>.
 //
+// ownCloud Infinite Scale for moodle Repository is build with the contribution of:
+// - Staatsinstitut für Schulqualität und Bildungsforschung
+// - JankariTech
+// - ownCloud - a kiteworks company.
```
Comment (phil-davis) [inline classes/configuration_exception.php:20]:
```suggestion
// - ownCloud - a Kiteworks company.
```

Is capital "K" used for Kiteworks?

File: classes/configuration_exception.php [hunk 2]
```diff
@@ -14,6 +14,11 @@
 // You should have received a copy of the GNU General Public License
 // along with the ocis repository for Moodle.  If not, see <http://www.gnu.org/licenses/>.
 //
+// ownCloud Infinite Scale for moodle Repository is build with the contribution of:
```
Comment (phil-davis) [inline classes/configuration_exception.php:17]:
```suggestion
// ownCloud Infinite Scale for moodle Repository is built with the contributions of:
```

**Review by phil-davis — COMMENTED** [review]
Minor suggested changes to be applied to all files.

File: classes/configuration_exception.php (hunk 1 above)
Comment (individual-it) [reply classes/configuration_exception.php]: done

File: classes/configuration_exception.php (hunk 2 above)
Comment (individual-it) [reply classes/configuration_exception.php]: done

**Review by phil-davis — APPROVED** [review]
Looks reasonable to me.
@dj4oC please review.

**Review by dj4oC — APPROVED** [review]
Straight forward. Like it. Thank you

**Pull request merged**